lstchain/scripts/lstchain_mc_dl2_to_irf.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python3
+
+"""
+Script to generate IRFs from MC DL2 files, using pyirf functions.
+This will save the IRFs, and either use fixed cuts or optimise them.
+and later it can be used by lstchain_dl2_to_dl3 to generate DL3 files.
+
+TODO: Add other IRFs
+Use optimised cuts and binning
+
+- Input: Path where the merged DL2 data HDF5 files are present
+- Output: IRFs compiled in fits.gz format
+
+Usage:
+$> python lstchain_mc_dl2_to_irf.py
```

Comment:
The name of the IRF is now hard coded in the file (irf.fits.gz). It would be great to have an option to set the name, eg --irf-file-name.

Reply:
I think I assumed it to be the standard name used. I will check if gammapy has any issue with any other name.

Reply:
No, there are no issues with a change of name. I have to change the name to have a different name for diffuse or point-like MC. And this is very convenient since in your script lstchain_dl2_to_dl3.py, you have the option --input-irf-file which allows to give the name of the input irf file. I do that and encounter no problem. I had to rename by hand the fits file, but would be more convenient to do it with an option in the script lstchain_mc_dl2_to_irf.py (and better also when everything will be automatic).

Reply:
Ok! I understand. I can change the output-irf-dir to output-irf-file, so that the user provides the name and path of the output file, like in lstchain_merge_hdf5_files.py. Later if there is a standardised nomenclature, it can be updated.